src/Tribe/Integrations/Freemius.php

```diff
@@ -65,7 +65,7 @@ public function __construct() {
 		 *
 		 * @param  bool  $should_load
 		 */
-		$should_load = apply_filters( 'tribe_events_integrations_should_load_freemius', true );
+		$should_load = apply_filters( 'tribe_events_integrations_should_load_freemius', $this->should_load( 10 ) );
```

Comment:
Do we need to pass the default value here?

Reply:
Just for future reference, we will change that int there in the future.